<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1098</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Radio</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="3">
     <widget class="QGroupBox" name="AttributeBox">
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="title">
       <string>Attribute Editor</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="1">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Radio</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="1">
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Translation</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="1">
              <widget class="QDoubleSpinBox" name="TranslateY">
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="minimum">
                <double>-99.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QDoubleSpinBox" name="TranslateX">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="minimum">
                <double>-99.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QDoubleSpinBox" name="TranslateZ">
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="minimum">
                <double>-99.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>Rotation</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="0" column="0">
              <widget class="QDoubleSpinBox" name="RotateX">
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="minimum">
                <double>-360.000000000000000</double>
               </property>
               <property name="maximum">
                <double>360.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QDoubleSpinBox" name="RotateZ">
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="minimum">
                <double>-360.000000000000000</double>
               </property>
               <property name="maximum">
                <double>360.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QDoubleSpinBox" name="RotateY">
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="minimum">
                <double>-360.000000000000000</double>
               </property>
               <property name="maximum">
                <double>360.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QGroupBox" name="groupBox_5">
            <property name="title">
             <string>Scale</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="0" column="1">
              <widget class="QDoubleSpinBox" name="ScaleY">
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="minimum">
                <double>-50.000000000000000</double>
               </property>
               <property name="maximum">
                <double>50.000000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QDoubleSpinBox" name="ScaleX">
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="minimum">
                <double>-50.000000000000000</double>
               </property>
               <property name="maximum">
                <double>50.000000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QDoubleSpinBox" name="ScaleZ">
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="minimum">
                <double>-50.000000000000000</double>
               </property>
               <property name="maximum">
                <double>50.000000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="4" column="1">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Camera</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0">
           <widget class="QGroupBox" name="groupBox_8">
            <property name="title">
             <string>Clipping Plane</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_11">
             <item row="0" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Near</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Far</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QDoubleSpinBox" name="NearCp">
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="value">
                <double>0.010000000000000</double>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QDoubleSpinBox" name="FarCp">
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="maximum">
                <double>10000.000000000000000</double>
               </property>
               <property name="value">
                <double>300.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QWidget" name="widget" native="true">
            <layout class="QGridLayout" name="gridLayout_9">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>FOV</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QDoubleSpinBox" name="CameraFOVInput">
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="minimum">
                <double>1.000000000000000</double>
               </property>
               <property name="value">
                <double>45.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QVBoxLayout" name="ToonSceneLayout">
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1098</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
